### DISCUSSION TITLE
(Question) How to load the KDE Plasma launcher on the niri session?

### DISCUSSION THREAD

**Discussion by hardBSDk** [YaLTeR/niri#74] [Q&A]

**Comment by YaLTeR** [comment]
What do you mean? Also, discussions should be a better place for questions like this.

**Comment by hardBSDk** [comment]
@YaLTeR I want to load the KDE Plasma taskbar after `cargo run --release`, it's possible to add the executable inside the niri config?

**Comment by YaLTeR** [comment]
If you just start it as is from a terminal, does it work? If it does, then you can add this to the config:
```
spawn-at-startup "name-of-executable"
```
